mail_debrand/i18n/it.po

```diff
@@ -6,38 +6,40 @@ msgid ""
 msgstr ""
```

Comment:
Why is this change here?

Reply:
Very strange, obviously .po file not necessary to modify. I will check my commit again and solve it.